src/components/app/data/queries/queryKeyFactory.js

```diff
@@ -90,6 +91,10 @@ const enterprise = createQueryKeys('enterprise', {
         queryKey: null,
         queryFn: async ({ queryKey }) => fetchEnterpriseCourseEnrollments(queryKey[2]),
       },
+      enterpriseGroupMemberships: (learnerEmail) => ({
+        queryKey: [learnerEmail],
+        queryFn: async ({ queryKey }) => fetchEnterpriseGroupMemberships(queryKey[2], learnerEmail),
```

Comment:
nit: do you think you could add a comment about what the different indices of queryKey are?

Reply:
I don't think it's needed since the query keys can be identified in the list.